bin/coreos-install

```diff
@@ -301,6 +301,14 @@ fi
 if [[ -z "${BASE_URL}" ]]; then
     BASE_URL="https://${CHANNEL_ID}.release.core-os.net/amd64-usr"
 fi
+
+# if the version is "current", resolve the actual version number
+if [[ "${VERSION_ID}" == "current" ]]; then
+    VERSION_ID=$(wget --inet4-only --quiet -O - "${BASE_URL}/${VERSION_ID}/version.txt" | \
+        gawk --field-separator '=' '/COREOS_VERSION_ID/ { print $2 }')
```

Comment:
For releases version and version ID are the same but on a technical level I actually name directories after the version so dev builds include the build ID.

</pedantery> I don't actually care which you use :P